heartbeat/process/monitor.go

```diff
@@ -298,6 +314,12 @@ func (m *Monitor) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedPe
 	return nil
 }
 
+// EpochConfirmed is called whenever an epoch is confirmed
+func (m *Monitor) EpochConfirmed(epoch uint32, _ uint64) {
+	m.flagHeartbeatDisableEpoch.SetValue(epoch >= m.heartbeatDisableEpoch)
+	log.Debug("heartbeat v1 monitor", "enabled", m.flagHeartbeatDisableEpoch.IsSet())
```

Comment:
wrong print. based on this PR, for epochs lower than 650 it will print:
`heartbeat v1 monitor enabled = false`
and for epochs higher than 650 it will print:
`heartbeat v1 monitor enabled = true`

which are quite the oposite